pkg/core/native/util.go

```diff
@@ -40,7 +40,8 @@ func setIntWithKey(id int32, dao *dao.Simple, key []byte, value int64) {
 	dao.PutBigInt(id, key, big.NewInt(value))
 }
 
-func getIntWithKey(id int32, dao *dao.Simple, key []byte) int64 {
+// GetIntWithKey retrieves integer by the specified contract ID and key.
+func GetIntWithKey(id int32, dao *dao.Simple, key []byte) int64 {
```

Comment:
Maybe this makes more sense in `dao`.

Reply:
I added DAO-level helper, but this one is also useful since it panics in case of missing SI.